modules/persistent-storage-rhv.adoc

```diff
@@ -0,0 +1,65 @@
+[id="persistent-storage-rhv_{context}"]
+= Creating a persistent volume on RHV
+
+When you create a PersistentVolumeClaim(PVC) object, OpenShift Container Platform provisions a new persistent volume (PV) and creates a PersistentVolume object.
+Prerequisites
+You are logged in to a running OpenShift Container Platform cluster.
+Correct RHV credentials in  “ovirt-credentials” secret.
+oVirt CSI Driver installed.
+At least one Storage class defined.
+Procedure (UI)
+To dynamically create a persistent volume on RHV using the web console:
+In the OpenShift Container Platform console, click Storage → Persistent Volume Claims.
+In the persistent volume claims overview, click Create Persistent Volume Claim.
+Define the required options on the resulting page.
+Select the appropriate StorageClass (ovirt-csi-sc by default).
+Enter a unique name for the storage claim.
+Select the access mode, currently RWO is the only supported access mode.
```

Comment:
what is RWO?

Reply:
ReadWriteOnce, as opposed to RWX ReadWriteMany option that was also included in the previous document